pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -597,6 +627,58 @@ func ExecuteLedCmd(ledName string) {
 	}
 }
 
+// createLogfile will use the arg to create a file
+func createLogfile(filename string) {
+	log.Functionf("createLogfile(%s)", filename)
+}
+
+// appendLogfile
+func appendLogfile(deviceNetworkStatus *types.DeviceNetworkStatus,
```

Comment:
Blindly appending seems like a pretty dangerous strategy to me. Why did you decide to just append @eriknordmark instead of re-writing?

Reply:
I only used the above to test that output was generated and the number of updates we get during a boot. The function isn't used (the filename is commented out for Qemu.*).
The real case will be to add another init and display function which invoke something like lcdproc or send directly on a serial port to an LCD.

Reply:
Ok, now I'm really confused @eriknordmark -- is GitHub not displaying for me correctly again -- here's what I see at the very top of your PR:

```
	{
		model:  "QEMU.*",
		regexp: true,
		// No disk light blinking on QEMU
		initFunc:    createLogfile,
		displayFunc: appendLogfile,
		// XXX set this to test output to a file:
		// arg:         "/persist/log/ledmanager-status.log",
		isDisplay: true,
	},
```

Reply:
Since arg has its default value of "", the appendLogfile call is no-op.